libraries/mediaplayer/impl/src/main/kotlin/io/element/android/libraries/mediaplayer/impl/MediaPlayerImpl.kt

```diff
@@ -76,12 +89,16 @@ class MediaPlayerImpl @Inject constructor(
     private val scope = CoroutineScope(Job() + Dispatchers.Main)
     private var job: Job? = null
 
-    private val _state = MutableStateFlow(MediaPlayer.State(false, null, 0L, 0L))
+    private val _state = MutableStateFlow(MediaPlayer.State(MediaPlayer.PlayState.Stopped, null, 0L, 0L))
 
     override val state: StateFlow<MediaPlayer.State> = _state.asStateFlow()
 
-    override fun acquireControlAndPlay(uri: String, mediaId: String, mimeType: String) {
+    override fun setMedia(uri: String, mediaId: String, mimeType: String, playWhenReady: Boolean) {
+        player.playWhenReady = false
```

Comment:
I don't understand why we would need to set play when ready as false here.

Reply:
I added this to make it explicit that calling `setMedia` will pause the playback straight away. I suppose `clearMediaItems` on the following line probably does this implicitly, so I can remove it if it doesn't add any value.